utils/generate-docs-examples/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "generate-docs-examples",
+  "version": "1.0.0",
+  "main": "generate-docs-examples.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
I realize this is the default package.json, but the license isn't ISC but Apache 2.0 and we don't need tests. Maybe all those lines can go away?

Reply:
Sorry, I need to fix this. ISC was added as a default when I ran `npm init -y` and I didn't notice. I'll change it to Apache.